op-chain-ops/cmd/ecotone-scalar/main_test.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"os/exec"
+	"testing"
+)
+
+func runMainWithArgs(t *testing.T, args []string) (string, error) {
+	t.Helper()
+	cmd := exec.Command("go", "run", "main.go")
+	cmd.Args = append(cmd.Args, args...)
+
+	var stdout, stderr bytes.Buffer
+	cmd.Stdout = &stdout
+	cmd.Stderr = &stderr
+
+	err := cmd.Run()
+	output := stdout.String() + stderr.String()
+	return output, err
+}
+
+func TestMain_PreEcotoneScalar(t *testing.T) {
+	output, err := runMainWithArgs(t, []string{"-decode=684000"})
```

Comment:
Let's add a test case for the OP Mainnet value as well.

Reply:
[d2b35d3](https://github.com/ethereum-optimism/optimism/pull/13472/commits/d2b35d34b081509d03779b97bc081300852790bd)